docs/topics/proc_using-a-nexus-repository-server-on-a-openshiftlocal.adoc

```diff
@@ -1,137 +1,157 @@
 // name variable defined locally, because it is only used in this topic
 :nexus-project-name: NEXUS_PROJECT_NAME
-// e.g. https://github.com/path/to/source.git
+// e.g.  https://github.com/path/to/source.git
 :scm-repo-url: SCM_REPOSITORY_URL
-[#using-a-nexus-repository-server-on-a-openshiftlocal]
-= Using a Nexus Repository Server on a {OpenShiftLocal}
+
+[id='using-a-nexus-repository-server-on-a-openshiftlocal_{context}']
+= Using a nexus repository server on a {OpenShiftLocal}
 
 While developing your cloud-native applications with Java and Maven you may be required to build them repeatedly.
 You can deploy a Nexus Repository server alongside the {launcher} tool on your {OpenShiftLocal} and use it to fetch artifacts from the Maven Central Repository and cache them locally.
 This helps you speed up your builds and rolling updates and alleviates network load during build time.
 
-== Prerequisites for Deploying Nexus on {OpenShiftLocal}.
-
+[discrete]
+== Prerequisites
 
-* Set up your {OpenShiftLocal}. For detailed steps on how to do this, follow the instructions in link:{link-launcher-openshift-local-install-guide}[{minishift-installation-guide-name}].
+* Your {OpenShiftLocal} set up.
+Follow the instructions in link:{link-launcher-openshift-local-install-guide}[{minishift-installation-guide-name}].
 
 ////
-* Set up your project for use with Maven. For more information on how to do this see the link:https://maven.apache.org/guides/getting-started/index.html[Maven Getting Started Guide^]
+* Set up your project for use with Maven.
+For more information on how to do this see the link:https://maven.apache.org/guides/getting-started/index.html[Maven Getting Started Guide^]
 ////
 
 ////
-* Verify that your application image has a `MAVEN_MIRROR_URL` environment variable. If it does not have this environmental variable, follow the link:https://books.sonatype.com/nexus-book/reference/config.html[Nexus documentation^] to configure your build.
+* Verify that your application image has a `MAVEN_MIRROR_URL` environment variable.
+If it does not have this environmental variable, follow the link:https://books.sonatype.com/nexus-book/reference/config.html[Nexus documentation^] to configure your build.
```

Comment:
Unrelated to this PR, but "environmental variable" is wrong, the previous sentence is correct, it's "environment variable".

Reply:
Fixed.